src/Switch/Switch.tsx

```diff
@@ -11,85 +11,14 @@ type Props = {
 
 export const Switch: React.FC<Props> = ({ disabled, id, currentValue, small, onChange }) => {
   return (
-    <div className={`toggle-switch${small ? ' small-switch' : ''}`}>
-      <input
-        type="checkbox"
-        className="toggle-switch-checkbox"
-        id={id}
-        checked={currentValue}
-        onChange={onChange}
-        disabled={disabled}
-      />
-      {id ? (
-        <label className="toggle-switch-label" htmlFor={id}>
-          <span
-            className={disabled ? 'toggle-switch-inner toggle-switch-disabled' : 'toggle-switch-inner'}
-            data-yes="yes"
-            data-no="no"
-          />
-          <span className={disabled ? 'toggle-switch-switch toggle-switch-disabled' : 'toggle-switch-switch'} />
-        </label>
-      ) : null}
-    </div>
+    <SwitchParent small={small}>
+      <Checkbox type="checkbox" id={id} checked={currentValue} onChange={onChange} disabled={disabled} />
+      {id && (
```

Comment:
Is this needed if `id` is required? Also, is `id` required?

Reply:
ID is required yeah, I will remove the conditional